test_utils/basetest.py

```diff
@@ -77,3 +79,63 @@ def get_git_info(self):
             branch_name = branch_name.decode()
 
         return repo_name, branch_name
+
+    def api_endpoint(self, sumo_deployment):
+        if sumo_deployment == "us1":
+            return "https://api.sumologic.com/api"
+        elif sumo_deployment in ["ca", "au", "de", "eu", "jp", "us2", "fed", "in"]:
+            return "https://api.%s.sumologic.com/api" % sumo_deployment
+        else:
+            return 'https://%s-api.sumologic.net/api' % sumo_deployment
+        
+    def create_collector(self, collector_name):
+        print("create_collector start")
+        collector_id = None
+        self.sumologic_cli = SumoLogic(os.environ["SUMO_ACCESS_ID"], os.environ["SUMO_ACCESS_KEY"], self.api_endpoint(os.environ["SUMO_DEPLOYMENT"]))
```

Comment:
move this line also in setup self.sumologic_cli = SumoLogic(...)
and in teardown self.sumologic_cli.session.close()

Reply:
Updated